ffs/scheduler/scheduler.go

```diff
@@ -211,6 +215,10 @@ func (s *Scheduler) Close() error {
 
 func (s *Scheduler) run() {
 	defer close(s.finished)
+	if err := s.resumeStartedDeals(); err != nil {
+		log.Errorf("resuming started deals: %s", err)
+		return
+	}
 	for {
```

Comment:
So before start listening to the new pushed `Jobs` to execute, we inspect if we have to resume deals that where orphaned from an interrupted close of Powergate. After we continued that and all clean, continue with the usual.